packages/core/src/components/overlay/overlay.tsx

```diff
@@ -309,10 +309,15 @@ export class Overlay extends React.Component<IOverlayProps, IOverlayState> {
     }
 
     private handleBackdropMouseDown = (e: React.MouseEvent<HTMLDivElement>) => {
-        if (this.props.canOutsideClickClose) {
-            safeInvoke(this.props.onClose, e);
+        const { backdropProps, canOutsideClickClose, enforceFocus, onClose } = this.props;
+        if (canOutsideClickClose) {
+            safeInvoke(onClose, e);
+        }
+        if (enforceFocus) {
```

Comment:
Failing so far. What I have:
```tsx
it("returns focus to overlay after clicking the backdrop if enforceFocus=true", () => {
    wrapper = mount(
        <Overlay enforceFocus={true} canOutsideClickClose={false} inline={true} isOpen={true}>
            {createOverlayContents()}
        </Overlay>,
    );
    wrapper.find(BACKDROP_SELECTOR).simulate("mousedown");
    assert.equal(document.body.querySelector(BACKDROP_SELECTOR), document.activeElement);
});
```
I can't figure out what the last line should be. As of now, it gives me this:
```
expected null to equal <body class="">
```
which I don't understand because the `BACKDROP_SELECTOR` element exists, also why is `document.activeElement` equals to `body` after mousing down on the backdrop?